### PR TITLE
docs: typos in production checklist

### PR DESCRIPTION
**Pull request by hifabienne**
- [ ] I am happy with the code
- [ ] Short description of the feature/issue is added in the pr description
- [ ] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.

### CODE REVIEW

File: docs/docs/self-hosting/manage/productionchecklist.md [hunk 1]
```diff
@@ -7,54 +7,54 @@ To apply best practices to your production setup we created a step by step check
 
 ### Infrastructure Configuration
 
-- [ ] make use of configmanagement such as Terraform to provision all of the below
-- [ ] use version control to store the provisioning
-- [ ] use a secrets manager to save your sensible informations
-- [ ] reduce the manual interaction with your platform to an absolute minimum 
+- [ ] Make use of configuration management such as Terraform to provision all of the below
```
Comment (fforootd) [inline docs/docs/self-hosting/manage/productionchecklist.md:10]:
```suggestion
- [ ] Make use of configuration management tools such as Terraform to provision all of the below
```

File: docs/docs/self-hosting/manage/productionchecklist.md [hunk 2]
```diff
@@ -7,54 +7,54 @@ To apply best practices to your production setup we created a step by step check
...
-- [ ] configure a valid [SMTP Server](/docs/guides/manage/console/instance-settings#smtp) and test emails
+- [ ] Configure a valid [SMTP Server](/docs/guides/manage/console/instance-settings#smtp) and test emails
 - [ ] Add [Custom Branding](/docs/guides/manage/customize/branding) if required
-- [ ] configure a valid [SMS Service](/docs/guides/manage/console/instance-settings#sms) such as Twilio if needed
-- [ ] configure your privacy policy, terms of service and a help Link if needed
-- [ ] secure your [masterkey](https://zitadel.com/docs/self-hosting/manage/configure)
-- [ ] declare and apply zitadel configuration using the zitadel terraform [provider](https://github.com/zitadel/terraform-provider-zitadel) 
+- [ ] Configure a valid [SMS Service](/docs/guides/manage/console/instance-settings#sms) such as Twilio if needed
+- [ ] Configure your privacy policy, terms of service and a help Link if needed
+- [ ] Secure your [masterkey](https://zitadel.com/docs/self-hosting/manage/configure)
```
Comment (fforootd) [inline docs/docs/self-hosting/manage/productionchecklist.md:40]:
```suggestion
- [ ] Keep your [masterkey](https://zitadel.com/docs/self-hosting/manage/configure) in a secure storage
```

File: docs/docs/self-hosting/manage/productionchecklist.md [hunk 3]
```diff
@@ -7,54 +7,54 @@ To apply best practices to your production setup we created a step by step check
...
+- [ ] Declare and apply zitadel configuration using the zitadel terraform [provider](https://github.com/zitadel/terraform-provider-zitadel) 
 
 ### Security
-- [ ] use a FQDN and a trusted valid certificate for external [TLS](/docs/self-hosting/manage/tls_modes#http2) connections
-- [ ] make use of different service accounts to secure ZITADEL within your hyperscaler or Kubernetes 
-- [ ] make use of a CDN service if needed to ease maintainability and firewall/DNS/WAF configuration
-- [ ] make use of a [security scanner](https://owasp.org/www-community/Vulnerability_Scanning_Tools) to test your application and cluster
+- [ ] Use a FQDN and a trusted valid certificate for external [TLS](/docs/self-hosting/manage/tls_modes#http2) connections
+- [ ] Make use of different service accounts to secure ZITADEL within your hyperscaler or Kubernetes
```
Comment (fforootd) [inline docs/docs/self-hosting/manage/productionchecklist.md:45]:
```suggestion
- [ ] Create service accounts for application that interacts with ZITADEL's APIs
```

File: docs/docs/self-hosting/manage/productionchecklist.md [hunk 4]
```diff
@@ -7,54 +7,54 @@ To apply best practices to your production setup we created a step by step check
...
+- [ ] Declare and apply zitadel configuration using the zitadel terraform [provider](https://github.com/zitadel/terraform-provider-zitadel) 
 
 ### Security
-- [ ] use a FQDN and a trusted valid certificate for external [TLS](/docs/self-hosting/manage/tls_modes#http2) connections
-- [ ] make use of different service accounts to secure ZITADEL within your hyperscaler or Kubernetes 
-- [ ] make use of a CDN service if needed to ease maintainability and firewall/DNS/WAF configuration
-- [ ] make use of a [security scanner](https://owasp.org/www-community/Vulnerability_Scanning_Tools) to test your application and cluster
+- [ ] Use a FQDN and a trusted valid certificate for external [TLS](/docs/self-hosting/manage/tls_modes#http2) connections
+- [ ] Make use of different service accounts to secure ZITADEL within your hyperscaler or Kubernetes 
+- [ ] Make use of a CDN service if needed to ease maintainability and firewall/DNS/WAF configuration
```
Comment (fforootd) [inline docs/docs/self-hosting/manage/productionchecklist.md:46]:
```suggestion
- [ ] Make use of a CDN service to decrease the load for static assets served by ZITADEL
```

File: docs/docs/self-hosting/manage/productionchecklist.md [hunk 5]
```diff
@@ -7,54 +7,54 @@ To apply best practices to your production setup we created a step by step check
...
+- [ ] Make use of a [security scanner](https://owasp.org/www-community/Vulnerability_Scanning_Tools) to test your application and cluster
```
Comment (fforootd) [inline docs/docs/self-hosting/manage/productionchecklist.md:47]:
```suggestion
- [ ] Make use of a [security scanner](https://owasp.org/www-community/Vulnerability_Scanning_Tools) to test your application and deployment environment
```

File: docs/docs/self-hosting/manage/productionchecklist.md [hunk 6]
```diff
@@ -7,54 +7,54 @@ To apply best practices to your production setup we created a step by step check
 
 ### Infrastructure Configuration
 
-- [ ] make use of configmanagement such as Terraform to provision all of the below
-- [ ] use version control to store the provisioning
-- [ ] use a secrets manager to save your sensible informations
-- [ ] reduce the manual interaction with your platform to an absolute minimum 
+- [ ] Make use of configuration management such as Terraform to provision all of the below
+- [ ] Use version control to store the provisioning
```
[the remark this answers is not in the document]
Comment (hifabienne) [reply docs/docs/self-hosting/manage/productionchecklist.md:11]: @thesephirot ping

File: docs/docs/self-hosting/manage/productionchecklist.md [hunk 7]
```diff
@@ -7,54 +7,54 @@ To apply best practices to your production setup we created a step by step check
 
 ### Infrastructure Configuration
 
-- [ ] make use of configmanagement such as Terraform to provision all of the below
-- [ ] use version control to store the provisioning
-- [ ] use a secrets manager to save your sensible informations
-- [ ] reduce the manual interaction with your platform to an absolute minimum 
+- [ ] Make use of configuration management tools such as Terraform to provision all of the below
+- [ ] Use version control to store the provisioning
```
Comment (fforootd) [inline docs/docs/self-hosting/manage/productionchecklist.md:11]:
```suggestion
```

**Review by fforootd — CHANGES_REQUESTED** [review]
@hifabienne I think removing the one line makes it ready to merge

**Review by fforootd — APPROVED** [review]

**Pull request merged**